crates/dprint/src/plugins/implementations/wasm/setup_wasm_plugin.rs

```diff
@@ -8,7 +8,7 @@ use crate::environment::Environment;
 
 use super::super::SetupPluginResult;
 
-pub const WASMER_COMPILER_VERSION: &str = "6.0.1";
+pub const WASMER_COMPILER_VERSION: &str = "6.1.0-rc.3";
```

Comment:
Thanks and sorry! I should update this and the tests to get the version automatically as part of the build.